### PR TITLE
Restores range autoselection in datepicker

### PR DESCRIPTION
**Pull request by shuwen**
Restores a bit of functionality missed in the 0.5 -> 1.0 port where the datepicker automatically picks a preset based on dates

review pls @dlasky @anthonykoerber 
cc @sassomedia

### CODE REVIEW

File: src/strand-datepicker/strand-datepicker.js [hunk 1]
```diff
@@ -237,18 +237,40 @@
...
+				}
+				return found;
+			}
+		},
+
+		_displayRange: function(newRange, oldRange) {
+			if(newRange !== oldRange) {
+				var range;
+				if(this._rangePresets)
+					range = this._rangePresets.filter(function(range) { return range.label === value })[0];
```
Comment (dlasky) [inline src/strand-datepicker/strand-datepicker.js]: `value` doesn't seem to be defined

Comment (shuwen) [inline src/strand-datepicker/strand-datepicker.js]: amended this commit, should be `newRange`

**Comment by dlasky** [comment]
👍

**Comment by anthonykoerber** [comment]
👍

**Comment by dlasky** [comment]
it passed in chrom--ignoring this stupid webdriver issue for the time being--you can merge it

**Pull request merged**